extensions/applicationinsights-analytics-js/Tests/Unit/src/TelemetryItemCreator.tests.ts

```diff
@@ -82,11 +91,166 @@ export class TelemetryItemCreatorTests extends AITestClass {
 
                 // assert
                 Assert.ok(telemetryItem);
-                Assert.equal("Microsoft.ApplicationInsights.{0}.Pageview", telemetryItem.name, "telemtryItem.name");;
+                Assert.equal("Microsoft.ApplicationInsights.{0}.Pageview", telemetryItem.name, "telemtryItem.name");
                 Assert.equal("PageviewData", telemetryItem.baseType, "telemetryItem.baseType");
                 Assert.deepEqual({"propKey1":"PropVal1","propKey2":"PropVal2"},telemetryItem.data, "telemetryItem.data");
             }
         });
+
+        
+        this.testCase({
+            name: "TelemetryItemCreatorTests: create a valid TelemetryItem for trackEvent with iKey",
+            test: () => {
+                // setup
+                const event: IEventTelemetry = {
+                    name: "trackEventNewtTest",
+                    iKey: "newTestIkey",
```

Comment:
Is there an existing test that checks if this is not provided (or is an empty string) that we then Assert that the "default" iKey is still populated?